template/gitlab-runner.tpl

```diff
@@ -14,9 +14,19 @@ then
   service docker start
 fi
 
-if [[ `echo ${runners_install_amazon_ecr_credential_helper}` == "true" ]]
-then
-  yum install amazon-ecr-credential-helper -y
+if [[ `echo true` == "true" ]]; then
```

Comment:
shouldn't this still be conditional on "runners_install_amazon_ecr_credential_helper", otherwise this if block might as well be removed

Reply:
Oops, yep, I think I copied this from a rendered template.